samples/SwiftDemoApp/SwiftDemoApp/GeoJSONViewController.swift

```diff
@@ -27,13 +27,14 @@ class GeoJSONViewController: UIViewController {
     mapView = GMSMapView.map(withFrame: CGRect.zero, camera: camera)
     self.view = mapView
 
-    let path = Bundle.main.path(forResource: "GeoJSON_sample", ofType: "json")
-    let url = URL(fileURLWithPath: path!)
-    geoJsonParser = GMUGeoJSONParser(url: url)
-    geoJsonParser.parse()
-
-    renderer = GMUGeometryRenderer(map: mapView, geometries: geoJsonParser.features)
-
-    renderer.render()
+    if let path = Bundle.main.path(forResource: "GeoJSON_sample", ofType: "json") {
```

Comment:
It's more Swift idiomatic to use `guard` here. Same comment in `KMLViewController`

i.e.

```swift
guard let path = Bundle.main.path(forResource: "GeoJSON_sample", ofType: "json") else {
    NSLog("Resource not available")
    return 
}
// ...
```

Reply:
Done. Thanks!